aiocache/factory.py

```diff
@@ -79,25 +88,98 @@ def create(self, alias=None, cache=None, **kwargs):
         cache = _create_cache(**{**config, **kwargs})
         return cache
 
-    def get_alias_config(self, alias):
-        config = self.get_config()
-        if alias not in config:
+    def get_alias_config(self, alias) -> MappingProxyType:
+        if alias not in self._config:
```

Comment:
minor: maybe better to use `self.config`

Reply:
It's just to avoid creation of MappingProxyType instance in property getter